pkg/reconciler/pipelinerun/resources/pipelinerunresolution.go

```diff
@@ -482,6 +568,7 @@ func GetPipelineConditionStatus(pr *v1beta1.PipelineRun, state PipelineRunState,
 	}
 
 	if reflect.DeepEqual(allTasks, withStatusTasks) {
+		// pipeline exits here with this condition
```

Comment:
Does this refer to the condition on the line above? 
I would not add this comment or I would put it above the condition.

Reply:
yes if check succeeds

Reply:
deleted